docs/references/protocol/data-types/permission-values.md

```diff
@@ -0,0 +1,84 @@
+With only the PermissionDelegation amendment, it's possible to assign permissions for transaction types that are reserved, unassigned, or part of amendments that are not currently enabled; it's also possible to assign PermissionValue `0` for full permissions. However, these values do not actually grant any permissions. This is a bug, and a future amendment will prevent assigning values outside of currently-enabled, delegatable transaction types or known granular permissions.
+{% /admonition %}
+
+## Granular Permissions
+[[Source]](https://github.com/XRPLF/rippled/blob/master/include/xrpl/protocol/detail/permissions.macro "Source")
+
+Granular Permissions have numeric types of 65537 and up, corresponding to specific names of permissions. Values that are not defined are not allowed. Each granular permission is a subset of a single transaction type's functionality.
+
+| Numeric Value | Name                     | Transaction Type       | Description |
+|:--------------|:-------------------------|:-----------------------|:------------|
+| `65537`       | `TrustlineAuthorize`     | [TrustSet][]           | Can [authorize individual trust lines](/docs/concepts/tokens/fungible-tokens/authorized-trust-lines). |
+| `65538`       | `TrustlineFreeze`        | [TrustSet][]           | Can [freeze individual trust lines](/docs/concepts/tokens/fungible-tokens/freezes). |
+| `65539`       | `TrustlineUnfreeze`      | [TrustSet][]           | Can [unfreeze individual trust lines](/docs/concepts/tokens/fungible-tokens/freezes). |
+| `65540`       | `AccountDomainSet`       | [AccountSet][]         | Can set the `Domain` field of the account. |
+| `65541`       | `AccountEmailHashSet`    | [AccountSet][]         | Can set the `EmailHash` field of the account. |
+| `65542`       | `AccountMessageKeySet`   | [AccountSet][]         | Can set the `MessageKey` field of the account. |
+| `65543`       | `AccountTransferRateSet` | [AccountSet][]         | Can set the [transfer fee of fungible tokens issued by the account](/docs/concepts/tokens/transfer-fees). |
+| `65544`       | `AccountTickSizeSet`     | [AccountSet][]         | Can set the [tick size of fungible tokens issued by the account](/docs/concepts/tokens/decentralized-exchange/ticksize). |
+| `65545`       | `PaymentMint`            | [Payment][]            | Can send payments that mint new fungible tokens or MPTs. |
+| `65546`       | `PaymentBurn`            | [Payment][]            | Can send payments that burn fungible tokens or MPTs. |
+| `65547`       | `MPTokenIssuanceLock`    | [MPTokenIssuanceSet][] | Can lock the balances of a particular MPT issued by the account. _(Requires the [MPTokensV1 amendment][] {% not-enabled /%}.)_ |
+| `65548`       | `MPTokenIssuanceUnlock`  | [MPTokenIssuanceSet][] | Can unlock the balances of a particular MPT issued by the account. _(Requires the [MPTokensV1 amendment][] {% not-enabled /%}.)_ |
+
+### Limitations to Granular Permissions
```

Comment:
I think we can remove this section. We already say in the previous section that granular permissions are hard-coded in the table above.

Reply:
This section does add a little bit more color and answers what I expect to be a common question, which is, "Can I grant permission to just send FOO and not XRP?" (No, you can't), so I'd like to keep it.